codecov.yml

```diff
@@ -1,2 +1,7 @@
 github_checks:
     annotations: false
+
+coverage:
```

Comment:
> We don't need such check in this SDK. Also, it can be false positive when moving from Java to Kotlin

Praise (❤️): I agree and thanks for this change!